c/validator.h

```diff
@@ -62,6 +66,57 @@ int csal_change_fetch(csal_change_t *state, const uint8_t key[CSAL_KEY_BYTES],
                       uint8_t value[CSAL_VALUE_BYTES]);
 void csal_change_organize(csal_change_t *state);
 
+#ifdef DEBUG_LOG
```

Comment:
Is there any chance we can either move this part to ckb-c-stdlib, or leverage existing parts in ckb-c-stdlib for debugging related code?

Reply:
Yes, I think it can be move to `ckb-c-stdlib`, need some prettify first.